.github/workflows/ci.yml

```diff
@@ -134,9 +134,9 @@ jobs:
               run: |
                   poetry run pytest backend/tests/
 
-            - name: Destroy AWS stack used for testing
+            - name: Destroy AWS stacks used for testing
               run: |
-                  poetry run cdk destroy --force geospatial-data-lake
+                  poetry run cdk destroy --force processing datalake  # don't drop networking
```

Comment:
If you're not going to drop networking it would be good to explain *why.*

Reply:
I added comment in c16a3329472d71183fdbf0dbaa4928125a8ac93a . I think this is best I can do. We don't have any "how to add new stack" docs and I don't think it is needed.